{"date":"Fri Jul 20 2018 10:18:13 GMT-0700 (PDT)","process":{"pid":68764,"uid":501,"gid":20,"cwd":"/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node","execPath":"/usr/local/bin/node","version":"v8.9.1","argv":["/usr/local/bin/node","/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node/index.js"],"memoryUsage":{"rss":59969536,"heapTotal":57044992,"heapUsed":20611200,"external":17867169}},"os":{"loadavg":[2.79296875,2.74169921875,2.95166015625],"uptime":575964},"trace":[{"column":11,"file":"/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node/startup/config.js","function":"module.exports","line":5,"method":"exports","native":false},{"column":28,"file":"/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node/index.js","function":null,"line":10,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":635,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":646,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":554,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":497,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":489,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":676,"method":"runMain","native":false},{"column":16,"file":"bootstrap_node.js","function":"startup","line":187,"method":null,"native":false},{"column":3,"file":"bootstrap_node.js","function":null,"line":608,"method":null,"native":false}],"stack":["Error: FATAL ERROR: jwtPrivateKey is not defined.","    at module.exports (/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node/startup/config.js:5:11)","    at Object.<anonymous> (/Users/moshfeghhamedani/Desktop/Trash/vidly-api-node/index.js:10:28)","    at Module._compile (module.js:635:30)","    at Object.Module._extensions..js (module.js:646:10)","    at Module.load (module.js:554:32)","    at tryModuleLoad (module.js:497:12)","    at Function.Module._load (module.js:489:3)","    at Function.Module.runMain (module.js:676:10)","    at startup (bootstrap_node.js:187:16)","    at bootstrap_node.js:608:3"],"level":"error","message":"uncaughtException: FATAL ERROR: jwtPrivateKey is not defined.","timestamp":"2018-07-20T17:18:13.911Z"}
{"date":"Thu Nov 18 2021 20:22:37 GMT+0200 (Eastern European Standard Time)","process":{"pid":60406,"uid":501,"gid":20,"cwd":"/Users/anasabufarraj/Development/vidly-api-node","execPath":"/opt/homebrew/Cellar/node/17.0.1/bin/node","version":"v17.0.1","argv":["/opt/homebrew/Cellar/node/17.0.1/bin/node","/Users/anasabufarraj/Development/vidly-api-node/index.js"],"memoryUsage":{"rss":69746688,"heapTotal":34095104,"heapUsed":20737264,"external":18914243,"arrayBuffers":17883318}},"os":{"loadavg":[2.07958984375,1.80859375,2.30029296875],"uptime":208604},"trace":[{"column":16,"file":"node:net","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1161,"method":"afterConnect [as oncomplete]","native":false},{"column":11,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":503,"method":null,"native":false},{"column":28,"file":"node:events","function":"Pool.emit","line":390,"method":"emit","native":false},{"column":12,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":326,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Connection.emit","line":390,"method":"emit","native":false},{"column":50,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":245,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Socket.emit","line":390,"method":"emit","native":false},{"column":8,"file":"node:internal/streams/destroy","function":"emitErrorNT","line":164,"method":null,"native":false},{"column":3,"file":"node:internal/streams/destroy","function":"emitErrorCloseNT","line":129,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":83,"method":null,"native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017","    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {","  name: 'MongoNetworkError'","}]","    at Pool.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js:503:11)","    at Pool.emit (node:events:390:28)","    at Connection.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js:326:12)","    at Object.onceWrapper (node:events:510:26)","    at Connection.emit (node:events:390:28)","    at Socket.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js:245:50)","    at Object.onceWrapper (node:events:510:26)","    at Socket.emit (node:events:390:28)","    at emitErrorNT (node:internal/streams/destroy:164:8)","    at emitErrorCloseNT (node:internal/streams/destroy:129:3)","    at processTicksAndRejections (node:internal/process/task_queues:83:21)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {\n  name: 'MongoNetworkError'\n}]","timestamp":"2021-11-18T18:22:37.368Z"}
{"date":"Fri Nov 19 2021 01:03:23 GMT+0200 (Eastern European Standard Time)","process":{"pid":63674,"uid":501,"gid":20,"cwd":"/Users/anasabufarraj/Development/vidly-api-node","execPath":"/opt/homebrew/Cellar/node/17.0.1/bin/node","version":"v17.0.1","argv":["/opt/homebrew/Cellar/node/17.0.1/bin/node","/Users/anasabufarraj/Development/vidly-api-node/index.js"],"memoryUsage":{"rss":67239936,"heapTotal":34095104,"heapUsed":20634456,"external":18914243,"arrayBuffers":17883318}},"os":{"loadavg":[2.15576171875,2.18115234375,2.05224609375],"uptime":225449},"trace":[{"column":16,"file":"node:net","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1161,"method":"afterConnect [as oncomplete]","native":false},{"column":11,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":503,"method":null,"native":false},{"column":28,"file":"node:events","function":"Pool.emit","line":390,"method":"emit","native":false},{"column":12,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":326,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Connection.emit","line":390,"method":"emit","native":false},{"column":50,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":245,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Socket.emit","line":390,"method":"emit","native":false},{"column":8,"file":"node:internal/streams/destroy","function":"emitErrorNT","line":164,"method":null,"native":false},{"column":3,"file":"node:internal/streams/destroy","function":"emitErrorCloseNT","line":129,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":83,"method":null,"native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017","    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {","  name: 'MongoNetworkError'","}]","    at Pool.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js:503:11)","    at Pool.emit (node:events:390:28)","    at Connection.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js:326:12)","    at Object.onceWrapper (node:events:510:26)","    at Connection.emit (node:events:390:28)","    at Socket.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js:245:50)","    at Object.onceWrapper (node:events:510:26)","    at Socket.emit (node:events:390:28)","    at emitErrorNT (node:internal/streams/destroy:164:8)","    at emitErrorCloseNT (node:internal/streams/destroy:129:3)","    at processTicksAndRejections (node:internal/process/task_queues:83:21)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {\n  name: 'MongoNetworkError'\n}]","timestamp":"2021-11-18T23:03:23.248Z"}
{"date":"Fri Nov 19 2021 18:47:29 GMT+0200 (Eastern European Standard Time)","process":{"pid":70148,"uid":501,"gid":20,"cwd":"/Users/anasabufarraj/Development/vidly-api-node","execPath":"/opt/homebrew/Cellar/node/17.0.1/bin/node","version":"v17.0.1","argv":["/opt/homebrew/Cellar/node/17.0.1/bin/node","/Users/anasabufarraj/Development/vidly-api-node/index.js"],"memoryUsage":{"rss":67551232,"heapTotal":33832960,"heapUsed":20752640,"external":19004355,"arrayBuffers":17973430}},"os":{"loadavg":[1.646484375,1.7197265625,1.994140625],"uptime":289295},"trace":[{"column":16,"file":"node:net","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1161,"method":"afterConnect [as oncomplete]","native":false},{"column":11,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":503,"method":null,"native":false},{"column":28,"file":"node:events","function":"Pool.emit","line":390,"method":"emit","native":false},{"column":12,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":326,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Connection.emit","line":390,"method":"emit","native":false},{"column":50,"file":"/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":245,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":510,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Socket.emit","line":390,"method":"emit","native":false},{"column":8,"file":"node:internal/streams/destroy","function":"emitErrorNT","line":164,"method":null,"native":false},{"column":3,"file":"node:internal/streams/destroy","function":"emitErrorCloseNT","line":129,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":83,"method":null,"native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017","    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {","  name: 'MongoNetworkError'","}]","    at Pool.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/topologies/server.js:503:11)","    at Pool.emit (node:events:390:28)","    at Connection.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/pool.js:326:12)","    at Object.onceWrapper (node:events:510:26)","    at Connection.emit (node:events:390:28)","    at Socket.<anonymous> (/Users/anasabufarraj/Development/vidly-api-node/node_modules/mongodb-core/lib/connection/connection.js:245:50)","    at Object.onceWrapper (node:events:510:26)","    at Socket.emit (node:events:390:28)","    at emitErrorNT (node:internal/streams/destroy:164:8)","    at emitErrorCloseNT (node:internal/streams/destroy:129:3)","    at processTicksAndRejections (node:internal/process/task_queues:83:21)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16) {\n  name: 'MongoNetworkError'\n}]","timestamp":"2021-11-19T16:47:29.371Z"}
